securityconfig/internal_users.yml

```diff
@@ -20,7 +20,7 @@ admin:
 kibanaserver:
   hash: "$2a$12$4AcgAt3xwOWadA5s5blL6ev39OXDNhmOesEoo33eZtrq2N0YrU3H."
   reserved: true
-  description: "Demo kibanaserver user"
+  description: "Demo opensearch dashboards server user"
```

Comment:
We are changing the description but keeping the role name as "kibana" itself which causes confusion. Is this expected ?

Reply:
Or should this PR be created at a later stage when roles are renamed also?

Reply:
If this is not a magic word, write "OpenSearch Dashboards" with camelcase spelling, "server" is not a thing AFAIK

Reply:
fixed